iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/posix_call.hpp

```diff
@@ -54,9 +52,6 @@ struct PosixCallResult
     int32_t errnum = POSIX_CALL_INVALID_ERRNO;
```

Comment:
@elfenpiff this is set to `-1` but it is never checked for this value. Shouldn't this be `0`, which is also the errno when no error occured?